### ISSUE TITLE
max_connections

### ISSUE THREAD

**Issue by mnietz** [sensu-plugins/sensu-plugins-postgres#16]
The plugin: check-postgres-connections.rb uses max_connections to measure the available client-connections. To be more precise you should use max_connections - superuser_reserved_connections as the amount of available connection for non-super-users.

```
show max_connection;
show superuser_reserved_connections;
```

```
select a.setting::int - b.setting::int as available_clients_connections from pg_settings a, pg_settings b where a.name = 'max_connections' and b.name = 'superuser_reserved_connections';
```

**Comment by eheydrick** [comment]
Good suggestion. I'd take a PR changing that.

**Comment by majormoses** [comment]
fixed in #34
